fastlane-plugin-revenuecat.gemspec

```diff
@@ -0,0 +1,36 @@
+lib = File.expand_path("lib", __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'fastlane/plugin/revenuecat/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'fastlane-plugin-revenuecat'
+  spec.version       = Fastlane::Revenuecat::VERSION
+  spec.author        = 'Toni Rico'
+  spec.email         = 'toni.rico.diez@revenuecat.com'
+
+  spec.summary       = 'A plugin including commonly used automation logic for RevenueCat SDKs.'
+  spec.homepage      = "https://github.com/RevenueCat/fastlane-plugin-revenuecat"
+  spec.license       = "MIT"
+
+  spec.files         = Dir["lib/**/*"] + %w(README.md LICENSE)
+  spec.require_paths = ['lib']
+
+  spec.required_ruby_version = '>= 2.5'
+
+  # Don't add a dependency to fastlane or fastlane_re
+  # since this would cause a circular dependency
+
+  # spec.add_dependency 'your-dependency', '~> 1.0.0'
+
+  spec.add_development_dependency('bundler')
+  spec.add_development_dependency('fastlane', '2.191.0')
```

Comment:
I had to downgrade fastlane to this version... If I run it with `2.192.0` or greater, I get the following error:
```
An error occurred while loading spec_helper. - Did you mean?
                    rspec ./spec/spec_helper.rb

Failure/Error: require 'fastlane' # to import the Action super class

Gem::ConflictError:
  Unable to activate fastlane-2.207.0, because optparse-0.2.0 conflicts with optparse (~> 0.1.1)
# ./spec/spec_helper.rb:12:in `<top (required)>'
# ------------------
# --- Caused by: ---
# LoadError:
#   cannot load such file -- fastlane
#   ./spec/spec_helper.rb:12:in `<top (required)>'
No examples found.
No examples found.
```

Reply:
I was able to fix this by downgrading my ruby for this project. Now I can use the latest fastlane version for this.